src/components/organisms/shared/OakPagination/OakPagination.test.tsx

```diff
@@ -71,14 +96,61 @@ describe("OakPagination Component", () => {
     expect(forwardsButton).toBeDisabled();
   });
 
+  it("handleChevronClick function", () => {
+    const onPageChangeMock = jest.fn();
+    const initialPage = 2;
+
+    const { getByTestId } = renderWithTheme(
+      <OakPagination
+        totalPages={5}
+        initialPage={initialPage}
+        onPageChange={onPageChangeMock}
+        paginationHref="/#"
+        pageName="test"
+      />,
+    );
+
+    const backwardsButton = getByTestId("backwards-button");
+    const forwardsButton = getByTestId("forwards-button");
+
+    fireEvent.click(backwardsButton);
+    expect(onPageChangeMock).toHaveBeenCalledWith(initialPage - 1);
+
+    fireEvent.click(forwardsButton);
+    expect(onPageChangeMock).toHaveBeenCalledWith(initialPage);
+  });
+
+  it("handleNumberClick function", () => {
+    const onPageChangeMock = jest.fn();
+    const initialPage = 2;
+
+    const { getAllByTestId } = renderWithTheme(
+      <OakPagination
+        totalPages={5}
+        initialPage={initialPage}
+        onPageChange={onPageChangeMock}
+        paginationHref="/#"
+        pageName="test"
+      />,
+    );
+
+    const numberButtons = getAllByTestId("page-number-component");
+
+    numberButtons.forEach((button, i) => {
+      fireEvent.click(button);
+      expect(onPageChangeMock).toHaveBeenCalledWith(i + 1);
+    });
```

Comment:
Just a word of caution about relying on the internals of the component to drive a test. If for whatever reason `getAllByTestId("page-number-component")` returned no elements this test will pass. 

I'd probably tack on an expectation that `numberButtons` returns exactly the number of elements you're expecting as a sanity check.

Reply:
Good shout, added the number expect statement for the number of `numberButtons` to expect also!